payments-core/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -102,6 +102,13 @@ class PaymentSheet internal constructor(
          * If set, PaymentSheet displays the button with this color.
          */
         var primaryButtonColor: ColorStateList? = null,
+
+        /**
+         * Whether or not your app can handle a delay between when the customer finishes paying and
+         * when your business receives money. Setting this to `true` allows your app to accept
+         * additional payment methods that may take time to move money (e.g. bank debits like SEPA).
```

Comment:
Is it worth saying that this will require webhooks and point to some documentation on how to set it up?

Reply:
This change is still under discussion, I just copied the description from iOS. This is a merge only into the `mc-EUL-July-2021` branch.

Reply:
The documentation will explain this in details, we can point to the documentation here.

Reply:
But also feel free to suggest an edit here.